python.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external python 2.7.6
+### RPM external python 2.7.10
```

Comment:
For what it's worth, many computing workflows utilize the curl module instead (which does enable validation) precisely because of the validation issue.  I'm a bit surprised to see that this is not true for the DAS client - maybe it's missing curl and falling back?

Anyhow, CERN ROOT CA tends to _not_ be available on the average system bundle (if it is, it's likely somewhat unique to the CERN site).